app/src/main/java/com/simply/birthdayapp/presentation/ui/screens/main/shops/details/ShopDetails.kt

```diff
@@ -0,0 +1,131 @@
+package com.simply.birthdayapp.presentation.ui.screens.main.shops.details
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.text.ClickableText
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.platform.LocalUriHandler
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.buildAnnotatedString
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.text.withStyle
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.simply.birthdayapp.R
+import com.simply.birthdayapp.presentation.models.Shop
+import com.simply.birthdayapp.presentation.ui.components.RoundAsyncImage
+import com.simply.birthdayapp.presentation.ui.theme.AppTheme
+
+@Composable
+fun ShopDetails(shop: Shop) {
```

Comment:
Name it like ShopDetailsComponent so it can be easily distinguished

Reply:
Fixed